tests/integration/replay/test_output.good

```diff
@@ -1,45 +1,94 @@
+makeAllocator <umpire::strategy::MixedPool>(host_mixedpool_default, getAllocator(HOST))
```

Comment:
Should the "good" output also be JSON?

Reply:
hmm..  I suppose it could even though it is not strictly required since it is only a normalized/sequenced view of operations which is only used by the test program (diff).

Should I spend the time to change that too?

Reply:
As discussed, leaving this in its current human readable form since it is easier to decipher diff failures if/when they fail.